functions/src/services/questionnaireResponse/fhirObservationConverter.ts

```diff
@@ -110,9 +111,5 @@ export function getNicotineObservationConfig(): QuestionnaireObservationConfig {
 }
 
 export function generateUUID(): string {
-  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
-    const r = (Math.random() * 16) | 0
-    const v = c === 'x' ? r : (r & 0x3) | 0x8
-    return v.toString(16).toUpperCase()
-  })
+  return randomUUID()
```

Comment:
why do you want to keep this function as a wrapper around the randomUUID() function rather than calling it directly and removing this? It doesn't add any value.